### ISSUE TITLE
Increase indent in pretty printed data structures to 4

### ISSUE THREAD

**Issue by boxed** [getsentry/sentry#12338]
The indent for nested dicts/lists/etc is set to two spaces. This is hard to follow. Increasing it to 4 would be much nicer.

**Comment by mattrobenolt** [comment]
? Can you provide any information, or an example?

**Comment by boxed** [comment]
I have a django project and you get local variables that are dicts outputted with 2 spaces indented. The pretty print is pretty ok otherwise.

**Comment by mattrobenolt** [comment]
Oh, sorry, you meant within our UI? I thought you were referring to the code style in our codebase here. :)

**Comment by boxed** [comment]
Yes, the GUI.

**Comment by BYK** [comment]
Closing this issue due to staleness. Feel free to comment here if you think we should still work on this.

**Comment by boxed** [comment]
Have you fixed it? Otherwise yes.

**Comment by BYK** [comment]
@boxed since this comes down to personal preferences and hasn't been echoed by anyone else for over a year now, I think we'll keep it to 2 spaces for now.

You may try to submit a PR if you are really passionate about this but I cannot guarantee the merge of it as this affects the design and may have other implications.
